moped-database/migrations/1665759117044_alter_table_public_moped_proj_personnel_alter_column_user_id/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."moped_proj_personnel" alter column "user_id" set not null;
+alter table "public"."moped_proj_personnel" alter column "project_id" set not null;
```

Comment:
we weren't enforcing it, but these columns should never be null. i need to double-check staging and prod to make sure these migrations will pass.